final_task/rss_reader/arguments.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+
+version = '5.0'
+
+""" Add argument commands for script """
+
+
+def command_line():
+    parser = argparse.ArgumentParser(
+        description='Pure Python command-line RSS reader.',
+        prog='rss-reader')
```

Comment:
Here and below: it is better to make this into one line, it will be still pretty readable.
Also it is better to use `f""` instead of `.format()`.